awx/settings/defaults.py

```diff
@@ -58,6 +58,13 @@ def IS_TESTING(argv=None):
     }
 }
 
+# Whether or not the deployment is a K8S-based deployment
+# In K8S-based deployments, instances have zero capacity - all playbook
+# automation is intended to flow through defined Container Groups that
+# interface with some (or some set of) K8S api (which may or may not include
+# the K8S cluster where awx itself is running)
+IS_K8S = False
```

Comment:
The intention is that this is set to `True` for *only* operator-based installs so that the capacity behavior is different.